roles/edpm_ovn/tasks/configure.yml

```diff
@@ -66,7 +66,7 @@
 
 - name: Configure OVS other_config
   ansible.builtin.shell: >
-    ovs-vsctl set open . {% for key, value in edpm_ovn_ovs_other_config.items() %} other_config:{{ key }}={{ value }} {% endfor %}
+    ovs-vsctl set open . {% for key, value in other_config.items() %} other_config:{{ key }}={{ value }} {% endfor %}
```

Comment:
need to change L61 instead so it works with edpm_enable_hw_offload=false:-
edpm_ovn_ovs_other_config: "{{ edpm_ovn_ovs_other_config | combine(hw_offload) }}"